src/assets/articles/sfscon-2024.md

```
@@ -0,0 +1,43 @@
+</div>
+
+SFSCon is a conference with a focus on free software and the open source community, with a strong history of 20 years. It was a great pleasure to be part of it!
+
+My talk was about <a href="https://www.sfscon.it/talks/optimizing-cloud-compute-resources-with-spare-cores/" target="_blank" rel="noopener">Optimizing Cloud Compute Resources with Spare Cores</a>, and as all standard talks, it was limited to 15 minutes .. for which I prepared over 100 slides, so it was a bit of a challenge, but managed to deliver on time — inlcuding a few minutes for questions at the end. If you are interested in a quick intro on what are the Spare Cores components, this might be a good starting point:
+
+<div style="padding:56.25% 0 0 0;position:relative;"><iframe src="https://player.vimeo.com/video/1030650515?badge=0&amp;autopause=0&amp;player_id=0&amp;app_id=58479" frameborder="0" allow="autoplay; fullscreen; picture-in-picture; clipboard-write" style="position:absolute;top:0;left:0;width:100%;height:100%;" title="SFSCON24 - Gergely Daroczi - Optimizing Cloud Compute Resources with Spare Cores"></iframe></div><script src="https://player.vimeo.com/api/player.js"></script>
+
+This was also the first time we had a booth at a conference, and we are super grateful for <a href="https://www.ow2.org/" target="_blank" rel="noopener">OW2</a> for offering us this opportunity to present our work to the community:
+
+<div class="text-center m-2.5 mt-8 mb-6">
+  <img class="zoomin w-full"
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)